package.json

```diff
@@ -44,6 +44,7 @@
     "firebase": "^9.22.2",
     "keycloak-js": "^21.1.1",
     "launchdarkly-js-client-sdk": "^3.1.3",
+    "moment": "^2.29.4",
```

Comment:
![image](https://github.com/bcgov/name-examination/assets/10002889/d2ffe224-de66-4679-bf96-ea4f3838a5a3)

Reply:
https://terodox.tech/migrating-away-from-momentjs-part1/

Reply:
https://www.reddit.com/r/javascript/comments/j9fq3n/momentjs_is_deprecated_heres_how_i_chose_a/

Reply:
Using Luxon now instead: https://www.npmjs.com/package/luxon

Reply:
Luxon looks great, but why do you need it when modern JS provides decent time/date functionality?

Reply:
I was considering to use either of them, but decided on luxon because: 
1) JavaScript's built-in Date object has limited support for time zone handling.
2) JavaScript doesn't provide built-in methods for formatting dates, and custom formatting needs to be created.

Reply:
Anyway, I'm OK with this if others are OK with it.